webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -394,17 +394,24 @@ export default {
       return this.activityId && this.activityId !== '';
     },
     postDisabled: function () {
-      return this.uploading || !this.news.title || !this.news.title.trim() || !this.news.body || !new DOMParser().parseFromString(this.news.body, 'text/html').documentElement.textContent.replace(/&nbsp;/g, '').trim();
+      if (new DOMParser().parseFromString(this.news.body, 'text/html').documentElement.querySelectorAll('img').length) {
+        return this.uploading || !this.news.title || !this.news.title.trim() || !this.news.body || !new DOMParser().parseFromString(this.news.body, 'text/html');
+      } else {
+        return this.uploading || !this.news.title || !this.news.title.trim() || !this.news.body || !new DOMParser().parseFromString(this.news.body, 'text/html').documentElement.textContent.replace(/&nbsp;/g, '').trim();
+      }
     },
     updateDisabled: function () {
       // disable update button while uploading an attachment
       if (this.uploading) {
         return true;
       }
       // disable update button if a mandatory field is empty
-      if (!this.news.title || !this.news.title.trim() || !this.news.body || !this.news.body.replace(/&nbsp;/g, '').trim()) {
+      if (!this.news.title || !this.news.title.trim() || !this.news.body.replace(/&nbsp;/g, '').trim() || new DOMParser().parseFromString(this.news.body, 'text/html').documentElement.querySelectorAll('img').length === new DOMParser().parseFromString(this.originalNews.body, 'text/html').documentElement.querySelectorAll('img').length && this.news.body === this.originalNews.body) {
```

Comment:
we can not rely on the number of images, as we can add and remove an image in the same time and it won't be considered as a modification of an article

Reply:
here is the a proposition
`      if (!this.news.title || !this.news.title.trim() || !this.news.body || !this.news.body.replace(/&nbsp;/g, '').trim()) {
        return true;
      }
      } else if (this.news.body.replace(/&nbsp;|<div>|<\/div>|<p>|<\/p>|\s+/g, '') !== this.originalNews.body.replace(/&nbsp;|<div>|<\/div>|<p>|<\/p>|\s+/g, '').replace(/\/>/g, '>').replace(/&#61;/g, '=')) {
        return false;
      } `